packages/sample-app-fabric/windows/SampleAppFabric/SampleAppFabric.cpp

```diff
@@ -90,27 +89,25 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   auto packageProviders{winrt::single_threaded_vector<winrt::Microsoft::ReactNative::IReactPackageProvider>()};
 
   RegisterAutolinkedNativeModulePackages(packageProviders);
-
   packageProviders.Append(winrt::make<CompReactPackageProvider>());
 
+  winrt::Microsoft::ReactNative::ReactViewOptions viewOptions;
+  viewOptions.ComponentName(mainComponentName);
+
   // Initialize and Manage the ReactNativeHost
   auto reactNativeAppBuilder{winrt::Microsoft::ReactNative::ReactNativeAppBuilder()
                                  .AddPackageProviders(packageProviders)
                                  .SetReactInstanceSettings(reactInstanceSettingsBuilder.ReactInstanceSettings())
                                  .SetAppWindow(window)
-                                 .SetCompositor(compositor)};
+                                 .SetReactViewOptions(viewOptions)};
 
   // Start the react-native instance by creating a javascript runtime and load the bundle.
-  auto host{reactNativeAppBuilder.Start()};
+  auto reactNativeWin32App{reactNativeAppBuilder.Build()};
 
-  // Create a RootView which will present a react-native component
-  winrt::Microsoft::ReactNative::ReactViewOptions viewOptions;
-  viewOptions.ComponentName(mainComponentName);
-  auto rootView = winrt::Microsoft::ReactNative::ReactNativeIsland(compositor);
-  rootView.ReactViewHost(winrt::Microsoft::ReactNative::ReactCoreInjection::MakeViewHost(host, viewOptions));
+  auto reactNativeIsland{reactNativeWin32App.ReactNativeIsland()};
```

Comment:
Everything from this line to Start should be in either Build or Start.

Reply:
As discussed, For version1 we are going to continue with this pattern,